lib/modules/versioning/swift/range.ts

```diff
@@ -49,29 +50,32 @@ function toSemverRange(range: string): string | null {
 }
 
 function getNewValue({ currentValue, newVersion }: NewValueConfig): string {
+  // Remove the v prefix if it exists
+  const cleanNewVersion = newVersion.replace(vPrefix, '$1');
+
   if (fromParam.test(currentValue)) {
-    return currentValue.replace(regEx(/".*?"/), `"${newVersion}"`);
+    return currentValue.replace(regEx(/".*?"/), `"${cleanNewVersion}"`);
   }
 
   const fromRangeMatch = fromRange.exec(currentValue);
   if (fromRangeMatch) {
     const [, version] = fromRangeMatch;
-    return currentValue.replace(version, newVersion);
+    return currentValue.replace(version, cleanNewVersion);
   }
 
   const binaryRangeMatch = binaryRange.exec(currentValue);
   if (binaryRangeMatch) {
     const [, , , version] = binaryRangeMatch;
-    return currentValue.replace(version, newVersion);
+    return currentValue.replace(version, cleanNewVersion);
   }
 
   const toRangeMatch = toRange.exec(currentValue);
   if (toRangeMatch) {
     const [, , version] = toRangeMatch;
-    return currentValue.replace(version, newVersion);
+    return currentValue.replace(version, cleanNewVersion);
   }
 
-  return currentValue;
+  return cleanNewVersion;
```

Comment:
this is a behavior change. why this now returns the new value instead of the current?

Reply:
It is. I believe that this was a bug. The logic was returning the current value instead of the new value.

Reply:
please do this change in a separate PR

Reply:
I reverted the logic fix.